### PR TITLE
Bug/consolidated bug fixes 201026

### PR DESCRIPTION
**Pull request by jamesatfish**
This PR implements the following tickets:

195339 - Sign out option not available on iOS
193596 - Descriptive text in settings is incorrect
195455 - Version info in settings page
196261 - Badge count on springboard icon does not auto-refresh
196512 - Favourited conversations create dot icon on conversations tab

This branch utilises the new LingoTests target introduced in PR #6 to specify unit tests for changes specific to the Lingo ACT Health project. Unit tests for 195339 and 196512 are included in this PR.

The branch is based upon act-master.

This branch will build on Xcode 12 / iOS 14.

### Pull Request Checklist

* [x] I read the [contributing guide](https://github.com/vector-im/element-ios/blob/develop/CONTRIBUTING.md)
* [x] UI change has been tested on both light and dark themes, in portrait and landscape orientations and on iPhone and iPad simulators
-Noting there is no change currently in the display behaviour between those two settings as per requirement.
* [x] Pull request is based upon act-master from the Pegacorn repo.
* [x] All new and existing unit tests pass successfully in this branch.

### CODE REVIEW

**Comment by jrschremmer** [comment]
Note 193596 should be 195396

File: Riot/Assets/it.lproj/Vector.strings [hunk 1]
```diff
@@ -358,7 +359,7 @@
 "settings_copyright" = "Copyright";
 "settings_term_conditions" = "Termini e condizioni";
 "settings_privacy_policy" = "Privacy Policy";
-"settings_acknowledgement" = "Acknowledgement";
+"settings_acknowledgement" = "Credits";
```
Comment (jrschremmer) [inline Riot/Assets/it.lproj/Vector.strings]: The change is also in en.lproj but not the other languages.  Is that intentional?

Comment (jamesatfish) [reply Riot/Assets/it.lproj/Vector.strings]: Updated the branch to revert the accidental change in the Italian file, and to make sure that the settings_acknowledgement string reference is in all language files with the original 'Acknowledgement' value.

File: Riot/Assets/en.lproj/Vector.strings [hunk 2]
```diff
@@ -507,7 +508,7 @@
 "settings_term_conditions" = "Terms & Conditions";
 "settings_privacy_policy" = "Privacy Policy";
 "settings_third_party_notices" = "Third-party Notices";
-"settings_acknowledgement" = "Acknowledgement";
+"settings_acknowledgement" = "Credits";
```
Comment (jrschremmer) [inline Riot/Assets/en.lproj/Vector.strings]: Does this change need to be reverted to?

Comment (jamesatfish) [reply Riot/Assets/en.lproj/Vector.strings]: Nope, that's an actual change for 195396 - "Acknowledgement setting should be called 'Credits'" (bug #3 in that ticket).

Comment (jrschremmer) [reply Riot/Assets/en.lproj/Vector.strings]: Ok then I'm confused why the 
"settings_acknowledgement" = "Credits"
change in the italian file was reverted back to:
"settings_acknowledgement" = "Acknowledgement"
and why the change from Acknowledgement -> Credits is not in all the other language files

Comment (jamesatfish) [reply Riot/Assets/en.lproj/Vector.strings]: Good questions, I just went back and checked the Element repo as I thought that settings_acknowledgement was an Element addition that hadn't been populated through the other language files correctly, hence the other files should have been updated with the "Acknowledgement" value to match the Element default, which was what that last commit was all about.

I was actually wrong, looks like settings_acknowledgement is specific to Lingo, so it makes sense for the string value across all translations to match our default option.

I've just pushed a new commit that not only updates the string value but updates the variable name itself to reflect our usage of the string in Lingo.

**Review by jrschremmer — APPROVED** [review]

**Pull request merged**